Testing/workflow/DeployUtils.Tests.ps1

```diff
@@ -0,0 +1,201 @@
+# The purpose of this test is to verify that some of the DeployUtils functions, used to publish ScubaGear, are working correctly.
+
+# Suppress PSSA warnings here at the root of the test file.
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidGlobalVars', '')]
+param()
+
+BeforeAll {
+  # Write-Warning $PSScriptRoot
+  . $PSScriptRoot/../../utils/DeployUtils.ps1
+}
+
+Describe "Check ConfigureScubaGearModule" {
+  It "The manifest file should be altered" {
+    # Get contents of manifest file.
+    $Location = Get-Location
+    $ManifestFilePath = Join-Path -Path $Location -ChildPath "PowerShell/ScubaGear/ScubaGear.psd1"
+    $OriginalContent = Get-Content -Path $ManifestFilePath
+    # ForEach ($Line in $OriginalContent) { Write-Warning $Line }
```

Comment:
```suggestion
```

Reply:
This is test code, and lines like this are for debugging.  It's handy to have these around for the next time I have to debug a problem.  I added a comment noting that this was for debugging, but I left the line there.